pyproject.toml

```diff
@@ -44,6 +44,8 @@ dependencies = [
     "click",
     "rich>=13.0.0",
     "rich-click",
+    # default litestar plugins
```

Comment:
Is it possible in PDM to move this to a different group called "litestar-plugins" (not bikeshedded),that is still mandatory but just for cosmetic reasons so its easier to read?